torch/distributed/rpc/constants.py

```diff
@@ -0,0 +1,5 @@
+from datetime import timedelta
+
+
+DEFAULT_NUM_SEND_RECV_THREADS = 4
```

Comment:
I think we should be clear that this is a arg specific to ProcessGroupAgent.

Putting these 2 in parallel is confusing users would assume they can change the number of threads for other RPC backends as well.

Reply:
This is done in https://github.com/pytorch/pytorch/pull/29068/